configs/zero123.yaml

```diff
@@ -25,12 +25,12 @@ data: # threestudio/data/image.py -> SingleImageDataModuleConfig
     eval_batch_size: 1
     elevation_range: [-10, 80]
     azimuth_range: [-180, 180]
-    camera_distance_range: [3.8, 3.8]
+    camera_distance_range: [3.8, 3.85]
```

Comment:
Do we want these changes to be permanent? Maybe add another yaml file with these changes called `configs/experimental/zero123_updated.yaml`?

Reply:
I believe the extra steps and the hyperparam changes are worth it in terms of higher quality... @voletiv do you mind testing it out with a few of your images?

Reply:
Actually... hold on, I want to run a few more tests